test/karma-conf.js

```diff
@@ -241,8 +241,9 @@ module.exports = function (config) {
                 'cd "' + process.cwd() + '" && npx tsc -p samples'
```

Comment:
You can save a bit performance and remove or comment out the test tool compiling further up on line 235-238.